CMakeLists.txt

```diff
@@ -190,13 +182,14 @@ Link options: ${CLP_FFI_JS_LINK_OPTIONS}."
         ${CLP_FFI_JS_BIN_NAME}
         SYSTEM
         PRIVATE
-        ${boost_SOURCE_DIR}
-        src/submodules/clp/components/core/src
-        src/submodules/clp/components/core/src/clp
-        src/submodules/clp/components/core/submodules
-        src/submodules/fmt/include
-        src/submodules/spdlog/include
-        src/submodules/zstd/lib
+        ${CLP_FFI_JS_BOOST_SOURCE_DIRECTORY}
+        ${CLP_FFI_JS_CLP_SOURCE_DIRECTORY}/components/core/src
+        ${CLP_FFI_JS_CLP_SOURCE_DIRECTORY}/components/core/src/clp
+        ${CLP_FFI_JS_FMT_SOURCE_DIRECTORY}/include
+        ${CLP_FFI_JS_NLOHMANN_JSON_SOURCE_DIRECTORY}/include
```

Comment:
added now that we can't recursively fetch CLP repo, which includes the json submomodule